src/main/java/com/rev_connect_api/config/WebConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.rev_connect_api.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class WebConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addResourceHandlers(ResourceHandlerRegistry registry) {
+        // Makes the resources/public directory accessible for images and videos
+        registry.addResourceHandler("/**")
+                .addResourceLocations("classpath:/public/");
```

Comment:
are we saving actual files here or meta data?

Reply:
Saving the actual file, not sure if it's the most ideal solution but that was one of the things I found online that could be used.